packages/CalendarCarousel/index.tsx

```diff
@@ -1,19 +1,269 @@
-import { StyleSheet, Text, View } from 'react-native';
+import {
+  FlatList,
+  SafeAreaView,
+  StyleSheet,
+  Text,
+  TextStyle,
+  TouchableOpacity,
+  View,
+  ViewStyle,
+} from 'react-native';
+import React, { ReactElement, useState } from 'react';
 
-import React from 'react';
+interface Style {
+  container: ViewStyle;
+  headerFlex: ViewStyle;
+  beforeArrowText: TextStyle;
```

Comment:
What about `arrowText`? They might have to look the same whether it is `left` or `right`.